src/encoded/batch_download.py

```diff
@@ -9,8 +9,8 @@
     parse_qs,
     urlencode,
 )
-from .search import iter_search_results
-from .search import list_visible_columns_for_schemas
+from encoded.search import iter_search_results
+from snowflakes.helpers.helper import list_visible_columns_for_schemas
```

Comment:
I don't like the dependency on snowflakes literally.   Maybe we should move them into snovault/helpers?

Reply:
Sure. I can do that.